common-util/Login/config.jsx

```diff
@@ -84,3 +85,29 @@ export const SUPPORTED_CHAINS_MORE_INFO = SUPPORTED_CHAINS.map((chain) => {
 
   return { id, networkDisplayName: name, networkName: getNetworkName() };
 });
+
+/**
+ * Solana supported chains
+ */
+export const SOLANA_SUPPORTED_CHAINS = [
+  {
+    id: null,
+    networkDisplayName: 'Solana',
+    networkName: 'solana',
+    blockchainName: 'solana',
```

Comment:
This naming is a bit inaccurate – the network and chain are basically synonymous. I think a more useful classifier is "vmType". I think from the frontend POV, the vm is the more relevant classifier.

Reply:
In general it's confusing that we use chain and network synonymously – really I think we should just go with one – probably network – when we're referring to something like Ethereum Mainnet, or Polygon zkEVM.

Reply:
Obviously the key should also update from solana to svm.